docs/user_manual/calculations.md

```diff
@@ -554,12 +563,100 @@ The assumptions used for calculations in power-grid-model are aligned to the one
+Please refer to their respective sections for detailed documentation.
+
+| Regulation type                                                   | Setting                                                                                 | Enum values                                                                 |
+| ----------------------------------------------------------------- | --------------------------------------------------------------------------------------- | --------------------------------------------------------------------------- |
+| [Automatic tap changing](#power-flow-with-automatic-tap-changing) | {py:meth}`tap_changing_strategy <power_grid_model.PowerGridModel.calculate_power_flow>` | {py:class}`TapChangingStrategy <power_grid_model.enum.TapChangingStrategy>` |
+
+#### Power flow with automatic tap changing
+
+```{warning}
+At the time of writing, this feature is still experimental and is not yet publicly available.
+```
+
+Some of the most important regulators in the grid affect the tap position of transformers.
+These {hoverxreftooltip}`user_manual/components:Transformer Tap Regulator`s try to regulate a control voltage $U_{\text{control}}$ such that it is within a specified voltage band.
+The $U_{\text{control}}$ may be compensated for the voltage drop during transport.
+Power flow calculations that take the behavior of these regulators into account may be toggled by providing one of the following strategies to the {py:meth}`tap_changing_strategy <power_grid_model.PowerGridModel.calculate_power_flow>` option.
+
+| Algorithm                                                    | Default  | Speed    | Algorithm call                                                                                              |
+| ------------------------------------------------------------ | -------- | -------- | ----------------------------------------------------------------------------------------------------------- |
+| No automatic tap changing (regular power flow)               | &#10004; | &#10004; | {py:class}`TapChangingStrategy.disabled <power_grid_model.enum.TapChangingStrategy.disabled>`               |
+| Optimize tap position for any value in the voltage band      |          | &#10004; | {py:class}`TapChangingStrategy.any_valid_tap <power_grid_model.enum.TapChangingStrategy.any_valid_tap>`     |
+| Optimize tap position for lower voltage in the voltage band  |          |          | {py:class}`TapChangingStrategy.min_voltage_tap <power_grid_model.enum.TapChangingStrategy.min_voltage_tap>` |
+| Optimize tap position for higher voltage in the voltage band |          |          | {py:class}`TapChangingStrategy.max_voltage_tap <power_grid_model.enum.TapChangingStrategy.max_voltage_tap>` |
+
+The following control logic is used.
+
+- Regulated transformers are ranked according to the amount of transformers between itself and the nearest source {hoverxreftooltip}`user_manual/components:source`.
```

Comment:
Maybe good to mention that they are ranked based on voltage level.

Reply:
what do you mean? they are regulated based on the weighted shortest path to the source, right? Or do you mean that only paths from the higher voltage to the same or lower rated voltage are taken into account?

Reply:
I meant that it is not necessarily the amount of transformers: if you have a transformer that has similar rated voltage on both sides, it does not count in the ranking process

Reply:
It does. Taken from the mathematical workout of the ranking algorithm:

>    2. For each non-transformer branch (link, line) which are connected at both sides 
>       1. Create two edges for both direction, with the weight zero. 
>    3. **For each transformer which are connected at both sides** 
>       1. **If there is no automatic tap changer** 
>          1. **Create two edges for both direction, with the weight one.** 
>       2. **If there is automatic tap changer** 
>          1. **Create one edge from the node at the tap side to the node without tap, with the weight one.** 

Lines are not taken into account, but all transformers are. The transformers with similar `u_rated` on both sides are just taken into account bidirectionally, rather than unidirectionally. That means that in the following grid

Reply:
cfr. discussion: indeed correct as is